src/components/AttachmentModal.js

```diff
@@ -212,10 +211,35 @@ function AttachmentModal(props) {
         [props.translate],
     );
     /**
-     * @param {Object} _file
+     * @param {Object} _data
+     * @returns {Boolean}
+     */
+    const isDirectoryCheck = useCallback(
+        (_data) => {
+            if (typeof _data.webkitGetAsEntry === 'function' && _data.webkitGetAsEntry().isDirectory) {
+                setIsAttachmentInvalid(true);
+                setAttachmentInvalidReasonTitle(props.translate('attachmentPicker.attachmentError'));
+                setAttachmentInvalidReason(props.translate('attachmentPicker.folderNotAllowedMessage'));
+                return false;
+            }
+            return true;
+        },
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+        [props.translate],
+    );
+    /**
+     * @param {Object} _data
      */
     const validateAndDisplayFileToUpload = useCallback(
-        (_file) => {
+        (_data) => {
+            if (!isDirectoryCheck(_data)) {
+                return;
+            }
+            /* eslint no-underscore-dangle: 0 */
```

Comment:
Why do we need to want to disable a lint rule here? Can we just rename the variable @Charan-hs ?

Reply:
Just to stay with the previous file name so. can I change that to `fileObject` @abdulrahuman5196  ?